crates/bevy_pbr/src/render/mesh_vertex_output.wgsl

```diff
@@ -0,0 +1,11 @@
+#define_import_path bevy_pbr::mesh_vertex_output
+
+@location(0) world_position: vec4<f32>,
+@location(1) world_normal: vec3<f32>,
+@location(2) uv: vec2<f32>,
+#ifdef VERTEX_TANGENTS
+@location(3) world_tangent: vec4<f32>,
+#endif
+#ifdef VERTEX_COLORS
+@location(4) colors: vec4<f32>,
```

Comment:
The variable only represents one colour, interpolated from the colour attributes of the vertices in the primitive. As such, I think `color` is a more suitable name.
```suggestion
@location(4) color: vec4<f32>,
```